notebooks/official/feature_store/sdk-feature-store.ipynb

```diff
@@ -1281,6 +1281,84 @@
         "After the LRO finishes, you should be able to see the result in the [BigQuery console](https://console.cloud.google.com/bigquery), as a new table under the BigQuery dataset created earlier."
       ]
     },
+    {
+     "cell_type": "markdown",
+     "metadata": {},
+     "source": [
+      "## Streaming Ingestion\n",
+      "\n",
+      "Streaming ingestion is currently public preview. \n",
+      "\n",
+      "Streaming ingestion lets you make real-time updates to feature values. While batch import is suitable for importing a large volume of data with high latency, streaming ingestion is suitable for ingesting small amount of data with low latency. The written data will be available to read using batch export and online serving. "
+     ]
+    },
+    {
+     "cell_type": "code",
+     "execution_count": null,
+     "metadata": {},
+     "outputs": [],
+     "source": [
+      "# Since streaming ingestion is public preview, the feature is in aiplatform_v1beta1.\n",
+      "from google.cloud import aiplatform_v1beta1\n",
+      "from google.api_core import operations_v1\n",
+      "from google.cloud.aiplatform_v1beta1 import FeaturestoreOnlineServingServiceClient\n",
+      "from google.cloud.aiplatform_v1beta1 import FeaturestoreServiceClient\n",
+      "from google.cloud.aiplatform_v1beta1.types import featurestore_online_service as featurestore_online_service_pb2\n",
+      "from google.cloud.aiplatform_v1beta1.types import types as types_pb2\n",
+      "from google.protobuf import field_mask_pb2\n",
+      "\n",
+      "API_ENDPOINT = \"{}-aiplatform.googleapis.com\".format(REGION)\n",
+      "# Create client connection\n",
+      "admin_client = FeaturestoreServiceClient(\n",
+      "    client_options={\"api_endpoint\": API_ENDPOINT})\n",
+      "data_client = FeaturestoreOnlineServingServiceClient(client_options={\"api_endpoint\": API_ENDPOINT})"
+     ]
+    },
+    {
+     "cell_type": "code",
+     "execution_count": null,
+     "metadata": {},
+     "outputs": [],
+     "source": [
+      "# Call write_feature_values to ingest data to users entity_type. \n",
```

Comment:
Should this be "to the user's entity_type" or "to the user-specified entity_type" or "to the specified entity_type"?

Reply:
Fixed